package.json

```diff
@@ -21,7 +23,25 @@
   },
   "homepage": "https://github.com/jdcj-oslabs/apimocking#readme",
   "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
```

Comment:
Looks like we have a mix of babel dependencies here—not sure I quite understand the difference between `@babel/…` and `babel-…` but either way some extra clarity here would be great.